### ISSUE TITLE
support backtrack path in export {^to-parent} bindings

### ISSUE THREAD

**Issue by whitecolor** [canjs/canjs#2132]
Hm it is working 2.3.4
http://jsbin.com/jonekemapo/1/edit?html,js,console,output

and not working on latest.
http://jsbin.com/jasuvokigu/1/edit?html,js,console,output

attribute should be exported to upper scopes using backtrack paths `{^selected-year}="../year"`

```HTML
<comp-one>
  {{#innerMap}}
  <year-selector  {^selected-year}="../year"></year-selector>      
  {{/innerMap}}  
  <br>on self: {{year}}<br>
  <br>on inner: {{innerMap.year}}<br>
</comp-one>
```

So `year` attribute be exported to `<comp-one>` `viewModel`.

**Comment by whitecolor** [comment]
@justinbmeyer When are you planing to fix latest version? 2.3.4 has some bugs (i.e. lacks this https://github.com/canjs/canjs/pull/2123) and 2.3.5 has another bugs that don't allow to use it.

**Comment by justinbmeyer** [comment]
@whitecolor I'm planning on working on it now.
